### PR TITLE
F/events

### PR DESCRIPTION
**Pull request by lRebornsl**
# 🎏   In this *Milestone*  🙋‍♂️⁉:
- ✅ Create a new directory called logic.
- ✅ Copy the files calculate.js and operate.js.
- ✅ Install **big.js** using NPM.
- ✅ Implement click event handlers using the math logic from calculate.js and operate.js.
- ✅ No linters error.
- ✅ Follow the best GitHub flow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,7 +83,7 @@ Install this project with:
 Example command:
 
 ```sh
- npx create-react-app .
+ npm install
 ```
```
Comment (1GORDON) [inline README.md:88]: Great job @lRebornsl 🤝

- [ ] Kindly fix the broken issues link to make your readme file more professional.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @lRebornsl ✋

Good job so far!

## Highlights!
- Props are passed correctly to any child components of the Calculate.js✔️ 
- `Calculator` app handles state with `useState` ✔️
- `click events` are implemented for all buttons ✔️
- `Calculator` component should import the `calculate.js` ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (lRebornsl) [reply README.md:88]: Solve

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @lRebornsl 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**